src/course/handbook/user-research.md

```diff
@@ -1,28 +1,14 @@
 # User Research & Usability Testing
```

Comment:
I do wonder if this should be in the design sprint section rather than the Handbook. Since they won't need it until ~week 7

Reply:
Yeah handbook is mostly meant to be generic stuff they'll need all the time, rather than week-specific resources. For the main curriculum everything just gets dumped in "workshops" and linked in the schedule.

Reply:
<img width="486" alt="Screenshot 2022-02-16 at 10 59 54" src="https://user-images.githubusercontent.com/42008435/154251114-ddfbeebd-287d-4b67-8637-ef26ad7ba1bd.png">
would this top bar in the design week schedule be a good place to list it @oliverjam ?

Reply:
I guess we could put this type of stuff in "Resources". I have historically just put external links in there, but no reason it can't be for our own stuff too

Reply:
I suggested to Reuben yesterday that we make a 'documentation' folder within projects and stick all the docs in there - linked from the schedule

Reply:
this was proving tricky with the navigation (since the schedule/learnings/spikes  top bar uses relative navigation instead of full links)

When I added a file inside a nested documentation folder, clicking the top bar links after navigating to `/in-house-design/documentation/user-research` wouldn't work as they would now look for `/in-house-design/documentation/projects` instead of `/in-house-design/projects` 



<img width="248" alt="Screenshot 2022-02-17 at 10 33 41" src="https://user-images.githubusercontent.com/42008435/154458107-9404135e-a1cf-4ff1-8d6c-04c661e0c2ce.png">

Since the top bar is shared across weeks I've just added the `user-research.md` document to the `in-house-design` folder and linked it in the resources file and also on the schedule.

<img width="311" alt="Screenshot 2022-02-17 at 10 36 37" src="https://user-images.githubusercontent.com/42008435/154458612-45e60f38-a9b8-440c-9359-f5fee3aea76c.png">

<img width="207" alt="Screenshot 2022-02-17 at 10 36 45" src="https://user-images.githubusercontent.com/42008435/154458726-d3b69389-dac8-459b-a946-a8ea28bedbcf.png">